### PR TITLE
Update sources.html.markdown

### PR DESCRIPTION
**Pull request by sprutner**
Moduels not updating was really annoying, should add this documentation in to increase usability of the feature.

### CODE REVIEW

File: website/source/docs/modules/sources.html.markdown [hunk 1]
```diff
@@ -107,6 +107,25 @@ module "consul" {
...
+
+You can also specify branches and version withs the ?ref query
+
+```hcl
+module "consul" {
+  source = "git::https://bitbucket.org/foocompany/module_name.git?hotfix
+}
+```
+
+You will need to run a `terraform get` -update=true if you want to pull the latest versions. This can be handy when you are rapidly iterating on a module in development.
```
Comment (grubernaut) [inline website/source/docs/modules/sources.html.markdown]: `-update=true` is outside of the syntax quotes.

Comment (sprutner) [inline website/source/docs/modules/sources.html.markdown]: No worries, edit away.

Comment (grubernaut) [inline website/source/docs/modules/sources.html.markdown]: @sprutner feel free to make the change, or I can. Want to give full credit where it's due 😄

**Comment by stack72** [comment]
I just fixed @grubernaut's issue - merging

**Pull request merged**